core-common/src/main/java/org/zalando/nakadi/annotations/validation/DeadLetterAnnotationValidator.java

```diff
@@ -0,0 +1,45 @@
+package org.zalando.nakadi.annotations.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.Map;
+
+public class DeadLetterAnnotationValidator implements
+        ConstraintValidator<DeadLetterValidAnnotations, Map<String, String>> {
+
+    public static final String AUTO_DLQ_FAILED_COMMIT_LIMIT = "nakadi.zalando.org/auto-dlq-failed-commit-limit";
```

Comment:
The name `auto-dlq` is not entirely correct, as we have only implemented skipping the events silently.
Should we change it to `auto-skip-event-failed-commit-limit` and make `dlq` a choice in another annotation value, e.g. `auto-skip-event-policy`?